LiverSegments/LiverSegments.py

```diff
@@ -40,6 +40,7 @@
 import os
 import unittest
 import logging
+import time
```

Comment:
Should we put this next to the call to time functions and make it only if Slicer is in developer mode?

[Here](https://github.com/ALive-research/Slicer-Liver/blob/3d0b17c812a865f67a4867ff2c8c11de07b7fe3b/LiverMarkups/Logic/vtkSlicerLiverMarkupsLogic.cxx#L131) I check whether Slicer is in developer mode in C++. I'm pretty sure in python is easy too